src/test/ui/resolve/issue-23305.stderr

```diff
@@ -1,15 +1,10 @@
-error[E0391]: cyclic dependency detected
+error[E0391]: cycle detected when processing `<impl at $DIR/issue-23305.rs:15:1: 15:20>`
   --> $DIR/issue-23305.rs:15:12
    |
 LL | impl ToNbt<Self> {}
-   |            ^^^^ cyclic reference
+   |            ^^^^
```

Comment:
Is there a reason to remove the label in the primary span?

Reply:
There isn't a reason to print it just on the first span, since it's a cycle. It should be on all the spans or none of the spans.

Reply:
Sadly, at the moment it is impossible to have span labels on anything other than the primary diagnostic. All note/help sections can only have spans without labels. (Not that it would improve _this_ diagnostic.)